app/controllers/metrics_controller.rb

```diff
@@ -0,0 +1,39 @@
+class MetricsController < ApplicationController
+
+  # searches in the description of the cases for keywords to order them in categories
+  def mapDescription(description)
+    health = ["hospital", "health", "medical", "psychologist", "pregnant"]
+    legal = ["legal", "interview appointment", "interview prep", "family reunification", "first decision", "second decision"]
```

Comment:
In the future we could also consider adding "tags" to cases, could quickly get out of hand but with a few good ones it could go far. But this is a great start!